sound/soc/sof/ipc4-topology.c

```diff
@@ -2008,9 +2024,6 @@ static int sof_ipc4_prepare_src_module(struct snd_sof_widget *swidget,
 		return output_format_index;
 	}
 
-	/* update pipeline memory usage */
-	sof_ipc4_update_resource_usage(sdev, swidget, &src->base_config);
-
```

Comment:
I don't understand how you moved this one since the base_config is used in the sof_ipc4_init_output_audio_fmt()??

If you can move it above, let's do it in a separate patch, then deal with the repeated part.